content/blog/eng/protected-friday.md

```diff
@@ -0,0 +1,126 @@
+---
+authors: ["engineering"]
+comments: true
+date: "2023-04-17"
+draft: false
+share: true
+categories: [Venerdì Protetto]
+title: "Venerdì Protetto"
+languageCode: "en-US"
+type: "post"
+twitterImage: '/images/logo_engineering.png'
+toc: false
+---
+
+# Venerdì Protetto
+
+## What is it all about?
+
+At Facile.it, we strongly believe in sharing knowledge and experiences. Thus, we dedicate one Friday a month to expanding our knowledge and learning among colleagues in the IT department. During these days called Venerdì Protetto (i. e. Protected Friday), we welcome speakers and participate in talks on IT-related topics. Speakers who bring to life Venerdì Protetto are colleagues or external guests.
+
+    
+    
+## Curious to learn more?
+
+Here you can find the talks of each Venerdì Protetto.
+
+<table>
+    <thead>
+        <tr
+            align="left"
+            >
+            <th>Date</th>
+            <th>Topic</th>
+            <th>Speakers</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan=4>14/4/2023</td>
+        </tr>
+        <tr>
+            <td><a href="#backstage">Backstage</a></td>
+            <td>Lee Mills</td>

```

Comment:
We should specify the company of each speaker, so we make clearer when it's internal or external.